R/theme-elements.r

```diff
@@ -491,7 +523,8 @@ el_def <- function(class = NULL, inherit = NULL, description = NULL) {
   plot.tag.position   = el_def("character"),  # Need to also accept numbers
   plot.margin         = el_def("margin"),
 
-  aspect.ratio        = el_def("character")
+  aspect.ratio        = el_def("character"),
+  geom                = el_def("element_geom", "geom")
```

Comment:
Is the `"geom"` argument in `el_def()` correct? Doesn't this imply that `geom` inherits from `geom`?